Makefile

```diff
@@ -7,7 +7,7 @@ VERSION            := $(shell git describe --tags --abbrev=0 2>/dev/null)
 LDFLAGS            :=
 
 # Test Repo is https://github.com/deis/empty-testbed
-TEST_REPO_COMMIT=589e15029e1e44dee48de4800daf1f78e64287c0
```

Comment:
This was accidentally broken in the rebase

Reply:
good catch